### PR TITLE
Bug 1446436 Edited rel notes and iptables followup

### PR DESCRIPTION
**Pull request by bfallonf**
As per: https://bugzilla.redhat.com/show_bug.cgi?id=1446436

and #4389 

Probably ready for a peer review @adellape @ahardin-rh @gaurav-nelson @bmcelvee @mburke5678 

Thanks, all!

### CODE REVIEW

File: admin_guide/iptables.adoc [hunk 1]
```diff
@@ -28,50 +30,49 @@ xref:../dev_guide/getting_traffic_into_cluster.adoc#using-nodeport[NodePorts].
...
-firewall managed by *iptables.service*. As with firewalld, *iptables.service*
-assumes total control of the iptables configuration, but unlike firewalld it does
-not provide any way for applications such as {product-title} to dynamically manage
-their own iptables rules. When *iptables.service* starts, it flushes and restores
-the complete iptables configuration. The restored rules are from its configuration
-file, *_/etc/sysconfig/iptables_*. The configuration file is not kept up to date
-during operation, so the dynamically added rules are lost during every restart.
+On older systems, or systems that do not support firewalld, your firewall might
+be managed by *iptables.service*. As with firewalld, *iptables.service* controls
+the iptables configuration, but unlike firewalld does not provide any way for
```
Comment (ahardin-rh) [inline admin_guide/iptables.adoc]: s/does not provide/it does not provide

File: admin_guide/iptables.adoc [hunk 2]
```diff
@@ -28,50 +30,49 @@ xref:../dev_guide/getting_traffic_into_cluster.adoc#using-nodeport[NodePorts].
 . Docker and some other software on the system may also use iptables
 rules for their own purposes.
 
-. Finally, administrators may want to add their own local rules to log
+. Administrators may want to add their own local rules to log
 or filter traffic.
```
Comment (ahardin-rh) [inline admin_guide/iptables.adoc]: s/may want to/can

**Comment by ahardin-rh** [comment]
@bfallonf Just minor suggestions from me. Looks good! :rocket:

**Comment by bfallonf** [comment]
Thanks @ahardin-rh ! I'm just going to follow up with a few other things and try get this merged.

**Comment by bfallonf** [comment]
Closing this PR. Turns out the thing to followup on was not supposed to happen.

**Pull request closed without merging**